mempool/clist_mempool.go

```diff
@@ -150,23 +149,24 @@ func (mem *cListMempool) recheckTxs(proxyAppConn proxy.AppConnMempool) {
 }
 
 func (mem *cListMempool) getNextTxBytes(remainBytes int64, remainGas int64, starter []byte) ([]byte, error) {
-	prevIdx, prevPriority := -1, uint64(math.MaxUint64)
+	prevPassed, prevElement := true, &clist.CElement{Priority: uint64(math.MaxUint64)}
```

Comment:
`prevElement` here makes me worried it may be treated as a real element. can you do something like

```golang
var prevElement *clist.CElement
if (starter) > 0 && e, ok := mem.txsMap.Load(TxKey(starter)); ok {
  prevElement = e.(*clist.CElement)
}

prevPriority := uint64(math.MaxUint64)
ignoreSamePriority := false
if prevElement != nil {
  prevPriority = prevElement.Priority
  ignoreSamePriority = true
}

```